.reuse/dep5

```diff
@@ -0,0 +1,4 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: sixtyfps
+Upstream-Contact: SixtyFPS GmbH <info@sixtyfps.io>
+Source: https://sixtyfps.io/
```

Comment:
What's that Source field?

Reply:
URL to some website where this thing comes from.

Reply:
`Source` is:

> Formatted text, no synopsis: an explanation from where the upstream source came from.

I could put in github if you think that is a better fit.